requirements.txt

```diff
@@ -133,7 +133,7 @@ sqlparse==0.4.4
     # via django
 typing-extensions==4.5.0
     # via asgiref
-urllib3==1.26.6
+urllib3==1.26.18
```

Comment:
The second link in the commit message points to a wrong CVE number. Should be CVE-2023-4**3**804.

Reply:
Will fix